aiidalab_widgets_base/structures.py

```diff
@@ -735,6 +736,45 @@ def _default_structure(self):
         return None
 
 
+def register_operator(operator):
```

Comment:
```suggestion
def _register_operator(operator):
```

I would recommend to make this explicitly private.

Reply:
I would prefer to keep it public as I would use it in the other editors.